ProjectGenerator/default.nix

```diff
@@ -9,5 +9,5 @@ stdenv.mkDerivation rec {
     cp bin/ihp-new $out;
     makeWrapper $out/ihp-new $out/bin/ihp-new --prefix PATH ":" "${git}/bin";
   '';
-  buildInputs = [ git makeWrapper ];
-}
\ No newline at end of file
+  buildInputs = [ git makeWrapper bash_5 ];
```

Comment:
I think this will not solve the issue as it's only defined as a build input here, but the bug is caused at runtime. For git we also use `makeWrapper` to make our compile time git available at runtime.

Reply:
So `bash` will also need to be wrapped with `makeWrapper`?

Reply:
Yes